Libs/Widgets/ctkFileDialog.cpp

```diff
@@ -67,6 +67,11 @@ void ctkFileDialogPrivate::init()
 {
   Q_Q(ctkFileDialog);
 
+  if (!(q->options() & QFileDialog::DontUseNativeDialog))
+  {
+    return;  // Native dialog does not supporting modifying or getting widget elements.
```

Comment:
I think @finetjul's suggestion was to not return but put the content that is only relevant for non-native dialog (which is currently all the content of this method) in the if branch. That way if some more features are added in the future there is smaller chance that it will be accidentally skipped.